2017-04-13T19:57:09.073534+00:00 app[web.1]: Traceback (most recent call last):
2017-04-13T19:57:09.073535+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 515, in spawn_worker
2017-04-13T19:57:09.073535+00:00 app[web.1]:     worker.init_process()
2017-04-13T19:57:09.073535+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 122, in init_process
2017-04-13T19:57:09.073536+00:00 app[web.1]:     self.load_wsgi()
2017-04-13T19:57:09.073537+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-04-13T19:57:09.073536+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 130, in load_wsgi
2017-04-13T19:57:09.073537+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-04-13T19:57:09.073538+00:00 app[web.1]:     self.callable = self.load()
2017-04-13T19:57:09.073538+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-04-13T19:57:09.073538+00:00 app[web.1]:     return self.load_wsgiapp()
2017-04-13T19:57:09.073539+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-04-13T19:57:09.073539+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-04-13T19:57:09.073539+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 357, in import_app
2017-04-13T19:57:09.073540+00:00 app[web.1]:     __import__(module)
2017-04-13T19:57:09.073540+00:00 app[web.1]:   File "/app/LMNOPproject/LMNOPsite/wsgi.py", line 14, in <module>
2017-04-13T19:57:09.073541+00:00 app[web.1]:     application = get_wsgi_application()
2017-04-13T19:57:09.073541+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
2017-04-13T19:57:09.073541+00:00 app[web.1]:     django.setup(set_prefix=False)
2017-04-13T19:57:09.073542+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/__init__.py", line 22, in setup
2017-04-13T19:57:09.073542+00:00 app[web.1]:     configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
2017-04-13T19:57:09.073543+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 56, in __getattr__
2017-04-13T19:57:09.073543+00:00 app[web.1]:     self._setup(name)
2017-04-13T19:57:09.073543+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 41, in _setup
2017-04-13T19:57:09.073544+00:00 app[web.1]:     self._wrapped = Settings(settings_module)
2017-04-13T19:57:09.073544+00:00 app[web.1]:     mod = importlib.import_module(self.SETTINGS_MODULE)
2017-04-13T19:57:09.073545+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/importlib/__init__.py", line 126, in import_module
2017-04-13T19:57:09.073544+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 110, in __init__
2017-04-13T19:57:09.073545+00:00 app[web.1]:     return _bootstrap._gcd_import(name[level:], package, level)
2017-04-13T19:57:09.073546+00:00 app[web.1]: ModuleNotFoundError: No module named 'lmn'
2017-04-13T19:57:09.073723+00:00 app[web.1]: [2017-04-13 19:57:09 +0000] [9] [INFO] Worker exiting (pid: 9)
2017-04-13T19:57:09.075632+00:00 app[web.1]: [2017-04-13 19:57:09 +0000] [8] [ERROR] Exception in worker process:
2017-04-13T19:57:09.075634+00:00 app[web.1]: Traceback (most recent call last):
2017-04-13T19:57:09.075634+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 515, in spawn_worker
2017-04-13T19:57:09.075635+00:00 app[web.1]:     worker.init_process()
2017-04-13T19:57:09.075636+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 122, in init_process
2017-04-13T19:57:09.075636+00:00 app[web.1]:     self.load_wsgi()
2017-04-13T19:57:09.075637+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 130, in load_wsgi
2017-04-13T19:57:09.075637+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-04-13T19:57:09.075638+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-04-13T19:57:09.075639+00:00 app[web.1]:     self.callable = self.load()
2017-04-13T19:57:09.075640+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-04-13T19:57:09.075640+00:00 app[web.1]:     return self.load_wsgiapp()
2017-04-13T19:57:09.075641+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-04-13T19:57:09.075642+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-04-13T19:57:09.075642+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 357, in import_app
2017-04-13T19:57:09.075643+00:00 app[web.1]:     __import__(module)
2017-04-13T19:57:09.075644+00:00 app[web.1]:   File "/app/LMNOPproject/LMNOPsite/wsgi.py", line 14, in <module>
2017-04-13T19:57:09.075645+00:00 app[web.1]:     application = get_wsgi_application()
2017-04-13T19:57:09.075645+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
2017-04-13T19:57:09.075646+00:00 app[web.1]:     django.setup(set_prefix=False)
2017-04-13T19:57:09.075646+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/__init__.py", line 22, in setup
2017-04-13T19:57:09.075647+00:00 app[web.1]:     configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
2017-04-13T19:57:09.075648+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 56, in __getattr__
2017-04-13T19:57:09.075648+00:00 app[web.1]:     self._setup(name)
2017-04-13T19:57:09.075667+00:00 app[web.1]:     self._wrapped = Settings(settings_module)
2017-04-13T19:57:09.075668+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 110, in __init__
2017-04-13T19:57:09.075666+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 41, in _setup
2017-04-13T19:57:09.075668+00:00 app[web.1]:     mod = importlib.import_module(self.SETTINGS_MODULE)
2017-04-13T19:57:09.075669+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/importlib/__init__.py", line 126, in import_module
2017-04-13T19:57:09.075669+00:00 app[web.1]:     return _bootstrap._gcd_import(name[level:], package, level)
2017-04-13T19:57:09.075670+00:00 app[web.1]: ModuleNotFoundError: No module named 'lmn'
2017-04-13T19:57:09.075671+00:00 app[web.1]: Traceback (most recent call last):
2017-04-13T19:57:09.075671+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 515, in spawn_worker
2017-04-13T19:57:09.075672+00:00 app[web.1]:     worker.init_process()
2017-04-13T19:57:09.075673+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 122, in init_process
2017-04-13T19:57:09.075673+00:00 app[web.1]:     self.load_wsgi()
2017-04-13T19:57:09.075674+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 130, in load_wsgi
2017-04-13T19:57:09.075674+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-04-13T19:57:09.075675+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-04-13T19:57:09.075676+00:00 app[web.1]:     self.callable = self.load()
2017-04-13T19:57:09.075676+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-04-13T19:57:09.075677+00:00 app[web.1]:     return self.load_wsgiapp()
2017-04-13T19:57:09.075677+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-04-13T19:57:09.075678+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-04-13T19:57:09.075679+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 357, in import_app
2017-04-13T19:57:09.075679+00:00 app[web.1]:     __import__(module)
2017-04-13T19:57:09.075680+00:00 app[web.1]:   File "/app/LMNOPproject/LMNOPsite/wsgi.py", line 14, in <module>
2017-04-13T19:57:09.075680+00:00 app[web.1]:     application = get_wsgi_application()
2017-04-13T19:57:09.075681+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
2017-04-13T19:57:09.075681+00:00 app[web.1]:     django.setup(set_prefix=False)
2017-04-13T19:57:09.075682+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/__init__.py", line 22, in setup
2017-04-13T19:57:09.075683+00:00 app[web.1]:     configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
2017-04-13T19:57:09.075683+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 56, in __getattr__
2017-04-13T19:57:09.075684+00:00 app[web.1]:     self._setup(name)
2017-04-13T19:57:09.075685+00:00 app[web.1]:     self._wrapped = Settings(settings_module)
2017-04-13T19:57:09.075686+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 110, in __init__
2017-04-13T19:57:09.075684+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 41, in _setup
2017-04-13T19:57:09.075686+00:00 app[web.1]:     mod = importlib.import_module(self.SETTINGS_MODULE)
2017-04-13T19:57:09.075687+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/importlib/__init__.py", line 126, in import_module
2017-04-13T19:57:09.075687+00:00 app[web.1]:     return _bootstrap._gcd_import(name[level:], package, level)
2017-04-13T19:57:09.075688+00:00 app[web.1]: ModuleNotFoundError: No module named 'lmn'
2017-04-13T19:57:09.075884+00:00 app[web.1]: [2017-04-13 19:57:09 +0000] [8] [INFO] Worker exiting (pid: 8)
2017-04-13T19:57:09.222595+00:00 app[web.1]: [2017-04-13 19:57:09 +0000] [4] [INFO] Shutting down: Master
2017-04-13T19:57:09.222676+00:00 app[web.1]: [2017-04-13 19:57:09 +0000] [4] [INFO] Reason: Worker failed to boot.
2017-04-13T19:57:09.315009+00:00 heroku[web.1]: State changed from up to crashed
2017-04-13T19:57:09.297172+00:00 heroku[web.1]: Process exited with status 3
2017-04-13T19:57:17.187195+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=powerful-refuge-24574.herokuapp.com request_id=8c07e59a-4d91-46ee-b82e-0f10750945a6 fwd="134.29.246.245" dyno= connect= service= status=503 bytes= protocol=https
2017-04-13T19:57:17.597759+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=powerful-refuge-24574.herokuapp.com request_id=b9f3976a-fde2-4763-84a4-840ac1523c3b fwd="134.29.246.245" dyno= connect= service= status=503 bytes= protocol=https
2017-04-13T19:57:35.908725+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=powerful-refuge-24574.herokuapp.com request_id=e71748f9-ce07-499b-9612-b2b7da07b6c2 fwd="134.29.246.245" dyno= connect= service= status=503 bytes= protocol=https
2017-04-13T19:57:36.327257+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=powerful-refuge-24574.herokuapp.com request_id=69d60380-6d48-4848-81e3-40f5b5bce93a fwd="134.29.246.245" dyno= connect= service= status=503 bytes= protocol=https
